cscs-checks/microbenchmarks/gpu/memory_bandwidth/memory_bandwidth.py

```diff
@@ -13,40 +13,20 @@ def __init__(self):
         self.valid_systems = ['daint:gpu', 'dom:gpu',
                               'arolla:cn', 'tsa:cn',
                               'ault:amdv100', 'ault:intelv100',
-                              'ault:amda100']
+                              'ault:amda100', 'ault:amdvega']
```

Comment:
I think that you are also missing a reference, don't you?

Reply:
No, sorry, I was confused. I thought you haven't put references for all the systems, but I see they are there.